e2e/framework/maestro/exec/exec.go

```diff
@@ -84,3 +84,25 @@ func ExecGoRun(dir string, env []string, externalArgs ...string) (*Cmd, error) {
 
 	return c, nil
 }
+
+func ExecRun(dir string, command string, args ...string) (*Cmd, error) {
```

Comment:
I would rename it to something like: ExecSysCmd.

Reply:
Also, there's a function called ExecGoRun, I would also change it for something like: ExecGoCmd

Reply:
c350ce2